src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -185,15 +192,14 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
 
     public async getPathToFile(fileName: string, directory: string) {
         const dirList = directory.split("/"); // Hardcoded as provisioning service is running on linux and we cannot use Path.sep as it is machine dependent
-        let directoryPath: string = "";
-        directoryPath = await this.localGitRepoHelper.getGitRootDirectory();
+        let directoryPath: string = this.tmpDirectoryPath;
         dirList.forEach((dir) => {
             try {
                 directoryPath = Path.join(directoryPath, dir);
                 // tslint:disable-next-line:non-literal-fs-path
```

Comment:
this rule does not seem to be practical. we should disable this rule in lint configuration.

Reply:
added in tslint